From 17966d27fe7e1ed8baabcef4328239f71053eef3 Mon Sep 17 00:00:00 2001
From: Dimitri Merejkowsky <dmerejkowsky@aldebaran-robotics.com>
Date: Tue, 27 Jan 2015 14:42:13 +0100
Subject: [PATCH] Install libraries into /usr/lib, not /usr/lib64

This causes a conflicts at installation. On Arch,
/usr/lib64 is a symlink to /usr/lib

This patch will no longer be necessary when
this CMake bug [1] is fixed

[1] http://www.cmake.org/Bug/view.php?id=15373
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 99f3648..cbb9f16 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -220,7 +220,7 @@ set(HAVE_DBUS BUILD_DBUS_SUPPORT)
 set(BIN_INSTALL_DIR          "${CMAKE_INSTALL_FULL_BINDIR}" CACHE PATH "The binaries directory")
 set(DATA_INSTALL_DIR         "${CMAKE_INSTALL_FULL_DATADIR}" CACHE PATH "The data directory")
 set(INCLUDE_INSTALL_DIR      "${CMAKE_INSTALL_FULL_INCLUDEDIR}" CACHE PATH "The subdirectory to the header prefix")
-set(LIB_INSTALL_DIR         "${CMAKE_INSTALL_FULL_LIBDIR}" CACHE STRING "Library directory name")
+set(LIB_INSTALL_DIR         "/usr/lib" CACHE STRING "Library directory name")
 set(PKGCONFIG_INSTALL_PREFIX "${LIB_INSTALL_DIR}/pkgconfig/" CACHE STRING "Base directory for pkgconfig files")

 # Where to install libraries and binaries
--
2.2.1

